app/views/licences/tabs/set-up.njk

```diff
@@ -10,135 +10,206 @@
   {% endfor %}
 {% endmacro %}
 
-{% set chargeVersionRows = [] %}
-
-{% set chargeVersionHead = [
-  {
-    text: "Start date"
-  },
-  {
-    text: "End date"
-  },
-  {
-    text: "Reason"
-  },
-  {
-    text: "Status"
-  },
-  {
-    text: "Action"
-  }
-] %}
-
-{% for chargeVersion in  chargeInformation %}
-  {% set chargeVersionRows = (chargeVersionRows.push([
-    {
-      text: chargeVersion.startDate
-    },
-    {
-      text: chargeVersion.endDate
-    },
-    {
-      text: chargeVersion.reason
-    },
-    {
-      html: statusTag(chargeVersion.status)
-    },
-    {
-      html: createLink(chargeVersion)
-    }
-  ]), chargeVersionRows) %}
-{% endfor %}
-
 <h2 class="govuk-heading-l">Licence set up</h2>
 
+  <h3 class="govuk-heading-m"> Requirements for returns </h3>
+
+  {% if returnVersions.length == 0 %}
+    <p class="govuk-body"> No requirements for returns for this licence.</p>
+  {% endif %}
+
+  {% if returnVersions.length > 0 %}
```

Comment:
You know what, just to be different I'm cool with an `else` here!

 ```suggestion
  {% else %}
```

And yes I should have spotted this when you added the other tables 🤦 😳

Reply:
crickey i missed that on all of them.